src/app/lib/utilities/describeDuration/index.test.js

```diff
@@ -0,0 +1,45 @@
+import moment from 'moment';
+import describeDuration from '.';
+
+const translationStub = {
+  h: 'Saa {x}',
+  hh: 'Masaa {x}',
+  m: 'Dakika {x}',
+  mm: 'Dakika {x}',
+  s: 'Sekunde {x}',
+  ss: 'Sekunde {x}',
+};
+
+const getDuration = seconds => {
+  return moment.duration(seconds, 'seconds');
+};
+
+const second = getDuration(1);
+const seconds = getDuration(20);
+const minute = getDuration(60);
+const minutes = getDuration(120);
+const hour = getDuration(3600);
+const hours = getDuration(7200);
+const day = moment.duration(25, 'hours');
+
+describe('describeDuration', () => {
+  it('returns the correct duration description', () => {
+    expect(describeDuration(minute)).toEqual('1 minute');
+    expect(describeDuration(minutes)).toEqual('2 minutes');
+    expect(describeDuration(second)).toEqual('1 second');
+    expect(describeDuration(seconds)).toEqual('20 seconds');
+    expect(describeDuration(hour)).toEqual('1 hour');
+    expect(describeDuration(hours)).toEqual('2 hours');
+    expect(describeDuration(day)).toEqual('25 hours');
+  });
```

Comment:
Could you add another test whose result is a combination of minutes+seconds?

Reply:
Tests added, all combinations tested. 👍